src/pages/TeamView/TeamView.js

```diff
@@ -35,6 +35,12 @@ const GET_TEAM_INFO = gql`
   }
 `;
 
+const SEND_INVITE_EMAILS = gql`
+  mutation inviteTeam($emails: [String!], $teamId: String!) {
+    inviteTeam(emails: $emails, teamId: $teamId)
+  }
+`;
+
```

Comment:
not your fault at all, but we should actually be putting gql mutations/queries somewhere abstracted. let's start now...

A good place might be
`src/data/gql/team/index.js`

i.e. inside the file you can just put 
```
const SEND_INVITE_EMAILS = gql`
  mutation inviteTeam($emails: [String!], $teamId: String!) {
    inviteTeam(emails: $emails, teamId: $teamId)
  }
`;
```